src/content/platform-integration/android/c-interop.md

```diff
@@ -1,45 +1,55 @@
 ---
-title: "Binding to native Android code using dart:ffi"
-description: "To use C code in your Flutter program, use the dart:ffi library."
+title: "Binding to native Android code"
+description: "To use C code in your Flutter program, use build hooks or the dart:ffi library."
 ---
 
 <?code-excerpt path-base="platform_integration"?>
 
-Flutter mobile and desktop apps can use the
-[dart:ffi][] library to call native C APIs.
-_FFI_ stands for [_foreign function interface._][FFI]
-Other terms for similar functionality include
-_native interface_ and _language bindings._
+This guide demonstrates how to bind your Flutter app to native Android code.
+
+## Overview
+
+Flutter mobile and desktop apps can use [build hooks][] or the the [dart:ffi][]
```

Comment:
build hooks also use `dart:ffi`, maybe it's better to

Flutter mobile and desktop apps can use the
[dart:ffi][] library to call native C APIs.
_FFI_ stands for [_foreign function interface._][FFI]
Other terms for similar functionality include
_native interface_ and _language bindings._
The FFI can be used in two ways:
1. via code assets reported in build hooks, in which Flutter takes care of bundling the native code, and
2. via legacy FFI plugins where you have to write the build files yourself.

(new bullet points)

Prefer using build hooks.